packages/components/package.json

```diff
@@ -16,8 +16,8 @@
     "test": "jest"
   },
   "peerDependencies": {
-    "react": "^16.13.1 || ^17.0.0",
-    "react-dom": "^16.13.1 || ^17.0.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
not support react@v17 yet 
https://github.com/ant-design/ant-design/issues/26136